miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/CallbackObj.kt

```diff
@@ -11,3 +11,21 @@ internal data class CallbackObj(
 
 @Keep
 internal data class Permission(val permission: String)
+
+@Keep
+internal data class CustomPermissionCallbackObj(
+    var action: String,
+    val param: CustomPermission?,
+    var id: String
+)
+
+@Keep
+internal data class CustomPermission(
+    val customPermissions: List<CustomPermissionObj>
```

Comment:
I think iOS changed this key to just `permissions` instead of `customPermissions`. Is that correct @ts-leojoseph-r ?

So this needs to be updated to match.

Reply:
Yes, as per the request 😄